avm/res/document-db/database-account/tests/e2e/kvSecrets/main.test.bicep

```diff
@@ -15,13 +15,14 @@ param resourceGroupName string = 'dep-${namePrefix}-documentdb.databaseaccounts-
 param resourceLocation string = deployment().location
 
 @description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
-param serviceShort string = 'dddasmi'
+param serviceShort string = 'dddaskvs'
 
 @description('Optional. A token to inject into the name of each resource.')
 param namePrefix string = '#_namePrefix_#'
 
 // Pipeline is selecting random regions which dont support all cosmos features and have constraints when creating new cosmos
-var enforcedLocation = 'eastus'
+var enforcedLocation = 'eastasia'
+var keyVaultName = 'dep-${namePrefix}-kv-${serviceShort}'
```

Comment:
For alignment, please move the `keyVautName` value to line 39 and reference it as an output from the `dependencies.bicep` file. This ensures that we have the dependency names in one place and ahve an implicit dependency configured.

Reply:
Done